.github/workflows/thank-you.yml

```
@@ -18,26 +18,20 @@ jobs:
-            const assignees_tagged = assignees.map((user) => '@' + user.login).join(' ')
-            const owner_tagged = '@' + user.login
+            const taggedUsers = [...new Set(
+              assignees.map(u => "@"+u.login).concat("@"+user.login)  
+            )].join(" ")
 
-            if (assignees.length == 0) {
-              await github.rest.issues.createComment({ ...options, body: `${owner_tagged} ${thankyouNote}` })
-            } else {
-              await github.rest.issues.createComment({ ...options, body: `${assignees_tagged} ${owner_tagged} ${thankyouNote}` })
-            }
+            await github.rest.issues.createComment({ 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)